libs/test-utils/src/expect_print.ts

```diff
@@ -16,6 +16,9 @@ let lastPrintedElementWithCallback: Optional<ElementWithCallback>;
 let lastPrintOptions: Optional<PrintOptions>;
 let errorOnNextPrint: Optional<Error>;
 
+let lastPdfPrintedElement: Optional<JSX.Element>;
+let lastPdfPrintOptions: Optional<PrintOptions>;
```

Comment:
Open to feedback on the best way to separate this. My thought was that printing to PDF and printing to paper are 2 distinct user actions so we should not have them share the same expectations ie. `expectPrint` should not succeed if the user prints to PDF.

Reply:
Seems reasonable. I know you're duplicating the existing code structure, but this doesn't feel like we need two separate bindings here. Could just be `lastPdfPrintCommand`.

Reply:
[73ea37b](https://github.com/votingworks/vxsuite/pull/3643/commits/73ea37b288f6ebd94d0d7438d026d4b6f0d7f95d)